src/App.tsx

```diff
@@ -7,41 +7,64 @@ import CodeEditor from './components/CodeEditor';
 import Settings from './components/Settings';
 import Header from 'layouts/Header';
 import Grid from './layouts/Grid';
+import Message from 'components/Message';
+import { ReactComponent as RequestEmpty } from './icons/requestEmpty.svg';
+import { ReactComponent as AllEmpty } from './icons/allEmpty.svg';
 
 function App() {
   const [current, send] = useMachine(coreMachine);
-  const { requests, resquestsMetaDataById, selectedRequest } = current.context;
+  const {
+    requests,
+    resquestsMetaDataById,
+    selectedRequest,
+    settings,
+  } = current.context;
 
   return (
     <div className="App">
       <Header
         title={'Requests'}
         Icon={<Settings send={send} current={current} />}
       />
-      <Grid
-        Left={
-          <Table
-            onRequestSelected={(request) =>
-              send({ type: 'OPEN_REQUEST_DETAILS', payload: { request } })
-            }
-            requests={requests}
-            resquestsMetaDataById={resquestsMetaDataById}
-            selectedRequest={selectedRequest}
-          />
-        }
-        Right={
-          current.matches('core.requestDetails') ? (
-            <CodeEditor
-              resquestMetaDataById={
-                resquestsMetaDataById[selectedRequest!.requestId]
+
+      {!Boolean(requests.length) ? (
+        <Message
+          Icon={<AllEmpty />}
+          message={
+            !Boolean(settings.urls.length)
+              ? "We're eager to start listening to your requests and help you to be more productive"
+              : 'Listening requests to: ' + settings.urls.join('\n')
```

Comment:
_Just to confirm, buddy_. This is the result of having two URLs and a short one as the first item:

![Screenshot_2020-04-15_09-22-37](https://user-images.githubusercontent.com/11931916/79336958-055b9300-7efb-11ea-88e1-a0ccbff9d953.png)

Should we also make sure we move the first item to a new line, for a better visualization? What do you think?

Reply:
yeah, totally! gonna work to improve it before we merge the PR